drivers/docker/utils.go

```diff
@@ -13,16 +13,25 @@ import (
 	"runtime"
 	"strings"
 
+	"github.com/distribution/reference"
 	"github.com/docker/cli/cli/config/configfile"
 	"github.com/docker/cli/cli/config/types"
-	"github.com/docker/distribution/reference"
 	registrytypes "github.com/docker/docker/api/types/registry"
 	"github.com/docker/docker/registry"
-	docker "github.com/fsouza/go-dockerclient"
 )
 
 func parseDockerImage(image string) (repo, tag string) {
-	repo, tag = docker.ParseRepositoryTag(image)
+	// deode the image tag
```

Comment:
```suggestion
	// decode the image tag
```

Also, oof, how is there not a `ParseImage` method with a structured output not already in [`github.com/distribution/reference`](https://pkg.go.dev/github.com/distribution/reference)? :crying_cat_face: I wonder if we could use the regex in that package to do this in a way that catches any edge cases in the surprisingly large grammar?